common/functions/src/scalars_v2/string.rs

```diff
@@ -39,5 +40,40 @@ pub fn register(registry: &mut FunctionRegistry) {
             Ok(())
         },
     );
+    registry.register_with_writer_1_arg::<StringType, NumberType<u64>, _, _>(
```

Comment:
`register_with_writer` is slower than `register_1_arg` I think.  Better to use the iterator which avoids too much extra reallocation.